inst/rmarkdown/templates/agu_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,229 @@
+---
+# Choose from this list of Journals:
+# JGR: Atmospheres, JGR: Biogeosciences, JGR: Earth Surface, 
+# JGR: Oceans, JGR: Planets, JGR: Solid Earth, JGR: Space Physics, 
+# Global Biogeochemical Cycles, Geophysical Research Letters, 
+# Paleoceanography and Paleoclimatology, Radio Science, Reviews of Geophysics, 
+# Tectonics, Space Weather, Water Resources Research, Geochemistry, Geophysics, 
+# Geosystems, Journal of Advances in Modeling Earth Systems (JAMES),
+# Earth's Future, Earth and Space Science, Geohealth
+journal: "JGR: Atmospheres"
+# Use draft to submit a paper
+options: "draft,linenumbers"
```

Comment:
I'd use `classoption`, following the convention of Pandoc: https://pandoc.org/MANUAL.html#variables-for-latex